cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -70,6 +79,8 @@
 
 os.environ['PATH'] += os.pathsep + os.path.join(os.sep, 'snap', 'bin')
 db = unitdata.kv()
+checksum_prefix = 'kubernetes-master.resource-checksums.'
+configure_prefix = 'kubernetes-master.prev_args.'
```

Comment:
Any chance you could change the prefixes to `kubernetes-common.resource-checksums.` and `kubernetes-common.prev_args.` so we don't have to pass them around everywhere? The upgrade-charm hook handlers on kubernetes-master and kubernetes-worker would need to migrate all the data over from the old prefixes, which is unfortunate, but the end result seems a lot cleaner.

Reply:
I don't have any strong concerns so I'm happy to defer to what you think is best. +1 to leaving it. :+1: